### ISSUE TITLE
broken website if enable dark reader + adguard extension or adguard windows

### ISSUE THREAD

**Issue by zhenjie1999** [AdguardTeam/AdguardBrowserExtension#1887]
### Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [yes] I am running the latest version
- [yes] I checked the documentation and found no answer
- [yes] I checked to make sure that this issue has not already been filed

### Issue Details
adguard extension problem:
enable adguard extension + darkreader on a webpage but the webpage turn into white color if adguard extension enabled
adguard windows problem:
https://bitsum.com/ become non stop loading page dead loop

pls check bottom Additional Information provided

- **AdGuard version:**
  - 3.6.14 download from edge store
- **Browser and version:**
  - edge 95.0.1020.53
- **Operating system and version:** 
  - windows 11 latest stable version
- **Filters you use in AdGuard:**
  - Activate the most appropriate filters automatically, Phishing and malware protection
  - AdGuard Base filter, AdGuard Annoyances filter, Filter unblocking search ads and self-promotion, AdGuard Japanese filter
- **Stealth Mode settings:**
  - no
- **Other browser extensions:**
  - 
![image](https://user-images.githubusercontent.com/75268245/142059018-8307e3af-892e-4ab4-8e17-e589484b6962.png)

### Expected Behavior

bug normally should display dark
![image](https://user-images.githubusercontent.com/75268245/142056753-701f0ac5-f31d-4280-842b-c1c71720e275.png)

### Actual Behavior

adguard disabled
![image](https://user-images.githubusercontent.com/75268245/142056846-5ff12cb3-d83b-4df9-9141-565dd84d9cc0.png)
dark reader setting
![image](https://user-images.githubusercontent.com/75268245/142057146-4da7ac34-8cb7-4f2e-b8ba-b0d5f887876c.png)

pls notice that this problem can only happen when right click google search result link and open webpage on new tab, only disable adguard extension and refreash webpage wont happen

below is my problem link
1. search from google
https://www.google.com/search?q=ORB%E3%81%A8phash&oq=ORB%E3%81%A8phash&aqs=edge..69i57.17862j0j1&sourceid=chrome&ie=UTF-8
2. open first result link
https://kzkohashi.hatenablog.com/entry/2018/01/05/205623

pls check this github report link https://github.com/darkreader/darkreader/issues/7336
### Screenshots

<details><summary>Screenshot:</summary>

</details>

### Additional Information
the github report link attachment is happen in adguard windows version, that one is more serious problem will cause website non stop loading dead loop

**Comment by Birbber** [comment]
Closing this issue as a duplicate of [#4000](https://github.com/AdguardTeam/AdguardForWindows/issues/4000).
